exporters/elasticsearch-exporter/src/test/java/io/camunda/zeebe/exporter/util/it/NonStartableBrokerRule.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.exporter.util.it;
+
+import io.camunda.zeebe.broker.system.configuration.BrokerCfg;
+import io.camunda.zeebe.broker.system.configuration.ExporterCfg;
+import io.camunda.zeebe.client.api.worker.JobHandler;
+import io.camunda.zeebe.client.api.worker.JobWorker;
+import io.camunda.zeebe.exporter.AbstractElasticsearchExporterIntegrationTestCase;
+import io.camunda.zeebe.exporter.api.Exporter;
+import io.camunda.zeebe.model.bpmn.BpmnModelInstance;
+import io.camunda.zeebe.protocol.record.Record;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Consumer;
+
+/**
+ * This interface is a temporary workaround to prevent actual tests from starting a broker manually.
+ * Instead, you should use {@link AbstractElasticsearchExporterIntegrationTestCase#startBroker()}.
+ *
+ * <p>As all of these utility classes will be removed by the end of Q2-2022, it's an OK trade-off to
+ * avoid rewriting most of the tests (which will be rewritten differently anyway).
+ */
+public interface NonStartableBrokerRule {
```

Comment:
:wrench: not sure why this was necessary since we any want to remove or rewrite this but ok :)

Reply:
Because while I dont think anyone will add a test, I can't guarantee it, so I just wanted to make sure you have to call the other method :smile: